setup.py

```diff
@@ -75,9 +76,10 @@ class PyTest(TestCommand):
     def initialize_options(self):
         TestCommand.initialize_options(self)
         self.pytest_args = [
-            'Tests',
+            'tests',
             '--cov=Robinhood/',
-            '--cov-report=term-missing'
+            '--cov-report=term-missing',
+	    '--pep8'
```

Comment:
This breaks?  Does pyflakes or a plugin need to be added to the `test_requires`?